### ISSUE TITLE
As a SO/CE, I would like content overview views to allow all sorting and have feature parity

### ISSUE THREAD

**Issue by jwfuller** [CuBoulder/express#1048]
## Context

`admin/content` and `content/blocks` should have feature parity and we should be able to sort by `Published` or by `Updated date`

**Comment by alexfinnarn** [comment]
`admin/content/blocks` does not have author, and `admin/content` cannot be sorted by updated date.

**Comment by cathysnider** [comment]
**2.7.0:** Content view sortable by Title, Type and Author; Block view sortable by Label and Title

**2.8.0:** Content view sortable by Title, Type, Author and Updated Date; Block view sortable by Label, Title, Author and Updated Date.
